Gems/Atom/RPI/Code/Source/RPI.Public/Culling.cpp

```diff
@@ -283,7 +283,7 @@ namespace AZ
             const Scene& scene,
             View& view,
             Frustum& frustum,
-            void* maskedOcclusionCulling)
+            [[maybe_unused]]void* maskedOcclusionCulling)
```

Comment:
nit: Missing a space before the type: `[[maybe_unused]]void*` -> `[[maybe_unused]] void*`

Reply:
Sorry, just saw this after merging the changes. Didn't know our coding standards called for a space after [[maybe_unused]] but will keep in mind for the future!